doc/ref/language.xml

```diff
@@ -539,9 +539,34 @@ Certain special facilities are provided for manipulating global
 variables which are not available for other types of variable (such as 
 local variables or function arguments).
 <P/>
-First, such variables may be marked <E>read-only</E>. In which case
+First, such variables may be marked <E>read-only</E> using
+<Ref Func="MakeReadOnlyGlobal"/>. In which case
 attempts to change them will fail. Most of the global variables
-defined in the &GAP; library are so marked.
+defined in the &GAP; library are so marked. <E>read-only</E> variables
+can be made read-write again by calling <Ref Func="MakeReadWriteGlobal"/>.
+GAP also features <E>constant</E> variables,
+which can never be changed. In some cases, GAP can optimise code which
+uses <E>constant</E> variables, as their value never changes.
```

Comment:
Hmm, the user at this point knows very little about constants, and has no idea how to learn more. E.g. they won't know that only immediate integers and some booleans can be made constant. I'd add a ref to `MakeConstantGVar` here, and then make sure they can learn enough about that in its documentation (which currently does not seem to exist...?).

Reply:
Getting back to this, it seems this this should be `MakeConstantGlobal` instead (also in the example below)